packages/plugin/src/Services/DiagnosticsService.php

```diff
@@ -170,306 +237,273 @@ public function getFreeformStats()
-                ]
-            ),
-            new DiagnosticItem(
-                'Craft Email configuration: <b>{{ value.transport }}</b>',
-                ['transport' => $emailTransport, 'issues' => $emailIssues],
-                [
-                    new SuggestionValidator(
-                        fn ($value) => 'misaligned_from' !== $value['issues'],
-                        'Potential Email Configuration issue',
-                        "When using SMTP for the Craft Email settings, the 'From Email' in email notification templates should always contain a matching email address, otherwise the notifications may not send. If you wish to have a different email address for this, consider using 'Reply to Email' instead."
-                    ),
-                    new NoticeValidator(
-                        fn ($value) => 'misaligned_from' !== $value['issues'],
-                        'Potential Email Configuration issue',
-                        "We've detected that you're using SMTP and have email notification template(s) that contain an email address for the 'From Email' that does not match the email address configured in the Craft Email settings."
-                    ),
-                ]
-            ),
-            'Spam settings' => [
-                new DiagnosticItem(
-                    'Spam Folder: [color]{{ value ? "Enabled" : "Disabled" }}[/color]',
-                    $this->getSummary()->statistics->spam->spamFolder,
-                    [
-                        new SuggestionValidator(
-                            fn ($value) => $value,
-                            'Enable built-in Spam Folder',
-                            'Freeform includes a built-in Spam Folder. It is beneficial for most sites to have this enabled to catch false positives due to spam configuration issues or rare cases. Freeform will let through spammy submissions but flag them as spam instead of blocking them outright. The benefit is that you can see false positives and learn why they were flagged as spam. You can also recover these and trigger proper email notifications, etc. <a href="{{ extra.url }}">Enable Spam Folder ></a>',
-                            ['url' => UrlHelper::cpUrl('freeform/settings/spam')]
-                        ),
-                        new NoticeValidator(
-                            fn ($value) => $value,
-                            '',
-                            'It is beneficial for most sites to have this enabled to catch false positives and/or look for patterns with spam.'
-                        ),
-                    ],
-                    fn ($value) => $value ? DiagnosticItem::COLOR_PASS : DiagnosticItem::COLOR_BASE
-                ),
+        foreach ($integrations as $integration) {
+            $name = $integration['name'];
+            $version = isset($integration['version']) && '' !== $integration['version'] ? "({$integration['version']})" : '';
```

Comment:
Rule of thumb here would be to always define a variable when having to lookup and use an array key more than once.

But in this case, you can just define the variable once, and then modify itself if the value is present;

Also, there's no need to do a `'' !==` check, since it will either be empty or it won't.

```suggestion
            $version = $integration['version'];
            $version = $version ? "({$version})" : ''
```